apps/web/src/components/contributors/contributors.tsx

```
@@ -0,0 +1,136 @@
+						<StatCard
+							title="Pull Requests"
+							value={prCount || 0}
+							description="Total PRs"
+							icon={GitPullRequest}
+							color="text-purple-500"
+							gradient="from-purple-500/10 to-pink-500/20"
+						/>
+					</div>
+
+					<div className="mx-auto w-full max-w-3xl">
+						<ActivityChart data={activityData!} />
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)